src/CashuWallet.ts

```diff
@@ -35,14 +35,13 @@ class CashuWallet {
 	 * @param proofs
 	 * @returns
 	 */
-	async checkProofsSpent(proofs: Array<Proof>): Promise<Array<Proof>> {
+	async checkProofsSpent(proofs: Array<Proof>, isSpendable = false): Promise<Array<Proof>> {
```

Comment:
is this so we can filter in both ways? if yes, maybe it would make sense to rename the function, since the function reads "check for spent proofs". I agree the naming is not great to begin with. But maybe the added complexity makes it even harder to understand what's going on. Or at the very least we could add it to the ts-doc comment, explaining the aditional param and update what the function returns. I'm leaning towards the latter, since we don't need to change exposed function signatures

Reply:
I agree. I will revert this change it may hurt more than it helps